src/Cosserat/mapping/LegendrePolynomialsMapping.h

```diff
@@ -1,58 +1,55 @@
-//
-// Created by younes on 17/11/2021.
-//
+/******************************************************************************
+*       SOFA, Simulation Open-Framework Architecture                          *
+*                (c) 2006-2018 INRIA, USTL, UJF, CNRS, MGH                    *
+*                                                                             *
+* This library is free software; you can redistribute it and/or modify it     *
+* under the terms of the GNU Lesser General Public License as published by    *
+* the Free Software Foundation; either version 2.1 of the License, or (at     *
+* your option) any later version.                                             *
+*                                                                             *
+* This library is distributed in the hope that it will be useful, but WITHOUT *
+* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or       *
+* FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License *
+* for more details.                                                           *
+*                                                                             *
+* You should have received a copy of the GNU Lesser General Public License    *
+* along with this library; if not, write to the Free Software Foundation,     *
+* Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301 USA.          *
+*******************************************************************************
+*                           Plugin Cosserat    v1.0                           *
+*				                                                              *
+* This plugin is also distributed under the GNU LGPL (Lesser General          *
+* Public License) license with the same conditions than SOFA.                 *
+*                                                                             *
+* Contributors: Defrost team  (INRIA, University of Lille, CNRS,              *
+*               Ecole Centrale de Lille)                                      *
+*                                                                             *
+* Contact information: https://www.inria.fr/fr/defrost                        *
+*                                                                             *
+******************************************************************************/
 #pragma once
-#include <Cosserat/config.h>
-#include <Cosserat/mapping/BaseCosseratMapping.h>
 
-#include <sofa/core/BaseMapping.h>
-#include <sofa/core/config.h>
-#include <sofa/core/Mapping.h>
-#include <sofa/defaulttype/SolidTypes.h>
-#include <sofa/defaulttype/RigidTypes.h>
+#include <Cosserat/config.h>
 #include <sofa/core/Mapping.h>
-#include <sofa/helper/ColorMap.h>
-
-#include <boost/math/special_functions/legendre.hpp>
+#include <sofa/core/trait/DataTypes.h>
```

Comment:
Is not compiling. Maybe miss something ?

Reply:
This is something that was merge 4 days ago: https://github.com/sofa-framework/sofa/pull/4788 so maybe you need a fresher sofa version.